runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -193,8 +196,10 @@ public boolean mergeSegments(Layout currentLayout)
 
         try {
             reconfigureSequencerServers(currentLayout, newLayout, true);
-        } catch (InterruptedException | ExecutionException e) {
-            log.error("mergeSegments: Bootstrapping sequencer failed due to exception : ", e);
+        } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)